src/Frontend/Path.hs

```diff
@@ -187,8 +187,8 @@ ideaMode (CommentIdea i)   root = root </> "idea" </> uriPart i </> "comment"
 ideaMode (OnComment ck m) root = commentMode ck m root
 ideaMode CreateIdea        root = root </> "idea" </> "create"
 
-commentAnchor :: IsString s => AUID Comment -> s
-commentAnchor (AUID c) = fromString $ "comment-" <> show c
+anchor :: IsString s => AUID a -> s
+anchor (AUID c) = fromString $ "comment-" <> show c
```

Comment:
What about "auid-" instead of "comment-" since this is no longer specific to comments.